### PR TITLE
Feat: pipdeptree

### LINKED ISSUE #334: Make git SHA and pipdeptree available as static files
Similar to e.g. `benefits.calitp.org/static/sha.txt`, it would be nice to be able to see the SHA of the deployed version of Eligibility Server as another quick check.

See also the work in cal-itp/benefits#1733 to get the Python dependency tree available as a static file in the image.

* [ ] Need to figure out how to serve static files from Eligibility Server -- ideally we aren't creating endpoints in the app, and this can be handled by `nginx` before the request even gets to the app (like how static files work in `benefits`).

### PR DESCRIPTION
**Pull request by angela-tran**
Closes #334 

Similar to cal-itp/benefits#1733

### CODE REVIEW

File: .github/workflows/docker-publish.yml [hunk 1]
```diff
@@ -22,6 +22,21 @@ jobs:
...
+          cache-dependency-path: "**/pyproject.toml"
+
+      - name: Write python packages to file
+        run: |
+          python -m venv .venv
+          source .venv/bin/activate
+          pip install pipdeptree
+          pip install -e .
+          pipdeptree
+          pipdeptree >> benefits/static/requirements.txt
```
Comment (angela-tran) [inline .github/workflows/docker-publish.yml:38]: ah need to change this

**Review by thekaveman — DISMISSED** [review]
Approving this, not sure why the other one failed though? 

https://github.com/cal-itp/eligibility-server/actions/runs/6342792432/job/17229278668

https://www.gnu.org/software/bash/manual/html_node/Redirections.html#Appending-Redirected-Output

> ... If the file does not exist it is created.

**Comment by angela-tran** [comment]
@thekaveman Hmm, maybe it'll create the file if it doesn't exist but not necessarily any parent directories. I'll test out this workflow more - gonna try running it locally with `act`!

**Comment by angela-tran** [comment]
I was able to reproduce the issue with `act` 😍 

image

I think this means we need to add a step to create the `static` directory so that it can have both `sha.txt` and `requirements.txt` in it.

For benefits we don't need this step since we already have a `static` directory with other stuff in it

**Comment by angela-tran** [comment]
With the step added in c3c010c, I got `act` to successfully write to `eligibility_server/static/sha.txt` and `eligibility_server/static/requirements.txt`, so hopefully it works in GitHub too!

image

**Review by thekaveman — APPROVED** [review]
Nice work! 🙏 `act`

**Pull request merged**